config/lib/app.js

```diff
@@ -46,6 +46,9 @@ module.exports.start = function start(callback) {
       if (process.env.NODE_ENV === 'secure') {
         console.log(chalk.green('HTTPs:\t\t\t\ton'));
       }
+      console.log(chalk.green('App version:\t\t\t' + config.meanjs.version));
```

Comment:
how about appVersion, and then we can do config.meanjs.meanVersion to avoid it being a string.